Package.swift

```diff
@@ -12,11 +12,19 @@
 
 import PackageDescription
 
-let targets: [PackageDescription.Target] = [
+var windowsPlatform: [Platform] = []
+#if os(Windows)
+windowsPlatform.append(.windows)
+#endif
```

Comment:
I'm not that familiar with manifests. Why is conditional compilation needed here?

Reply:
🤔 AFAIK this is used to comfort SwiftPM 5.3 because it doesn't recognize Windows.  This workaround is introduced by @compnerd for other projects and I just use it here.

Reply:
I don't think that this would be needed given the comment below.